framework/console/controllers/MigrateController.php

```diff
@@ -186,6 +187,12 @@ protected function createMigration($class)
         if (strpos($class, '\\') === false) {
             $file = $this->migrationPath . DIRECTORY_SEPARATOR . $class . '.php';
             require_once($file);
+        } elseif (!class_exists($class, false)) {
+            // detect aliases migration in global namespace
+            require_once(Yii::getAlias('@'.str_replace('\\', '/', $class)).'.php');
```

Comment:
this step is done by the Yii autoloader in case you enable second parameter in `class_exists`, so it is not needed here.